components/TransactionCard/TransactionCard.styled.js

```diff
@@ -54,14 +75,64 @@ export const TransactionDate = styled.p`
   font: 400 12px/14px var(--font-family);
 `;
 
-export const TransactionAmount = styled.p`
+export const ActionLinkContainer = styled.div`
+  flex: 3;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+export const ActionLink = styled(Link)`
+  text-decoration: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 30px;
+  width: 30px;
+  color: black;
+`;
+
+export const ActionButton = styled.button`
+  text-decoration: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 30px;
+  width: 30px;
+  border: none;
+  background-color: transparent;
+  padding: 0;
+  cursor: pointer;
+`;
+
+export const ActionInfo = styled(IoInformationCircleOutline)`
+  height: 26px;
+  width: 26px;
+`;
+
+export const ActionDetails = styled(TiEdit)`
+  height: 25px;
+  width: 25px;
+`;
+
+export const ActionDelete = styled(FaRegTrashCan)`
+  height: 21px;
```

Comment:
e.g. you could use aria-labels 'aria-label': 'Delete' and roles `role`: `button` here and for the other Icons